src/IniFile.cpp

```diff
@@ -49,6 +49,9 @@ cIniFile::cIniFile(void) :
 
 bool cIniFile::ReadFile(const AString & a_FileName, bool a_AllowExampleRedirect)
 {
+
+	m_Filename = a_FileName;
+
```

Comment:
And I really don't like how much the code is dependent of Inifiles. What if we want to change the config file? The way I view WriteFile, is that it is a separate dump to this file, don't change the backing store.